spec/commands/decidim/half_signup/send_verification_spec.rb

```diff
@@ -27,6 +28,10 @@
 
   before do
     allow(SecureRandom).to receive(:random_number).and_return(verification)
+
+    Decidim.configure do |config|
```

Comment:
Prefer to define mock for `Decidim.configure.sms_gateway_service` 

Example : `allow(Decidim.config.sms_gateway_service).to receive(:present?).and_return(true)`